app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -171,4 +189,70 @@ private void deleteStudentImagesRecursive(StudentImage studentImage, String reas
         studentImageDao.delete(studentImage);
         Log.i(getClass().getName(), "StudentImage with the id " + studentImage.getId() + " has been deleted because " + reason);
     }
+
+    public synchronized void trainClassifier(){
+        // Initiate training if a StudentImageCollectionEvent has not been trained yet
+        if (studentImageCollectionEventDao.queryBuilder().where(StudentImageCollectionEventDao.Properties.SvmTrainingExecuted.eq(false)).count() > 0){
```

Comment:
Save  `studentImageCollectionEventDao.queryBuilder().where(StudentImageCollectionEventDao.Properties.SvmTrainingExecuted.eq(false)).count()` as a variable, and print it to the log output.

E.g: `int count = studentImageCollectionEventDao.queryBuilder().where(StudentImageCollectionEventDao.Properties.SvmTrainingExecuted.eq(false)).count();`